readme.md

```diff
@@ -124,5 +156,9 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
+One technique for a team would be for one person to initialize the repo, and then each team member forks a copy, clones it to their own desktop, makes changes, pushes them back up to their forked copy and then makes a pull request to the parent repo, which a team member would review and then merge with master.
+
+Another technique is for the team to use one remote repo to clone down to each machine, but then each member would create their own branches to make changes, and would then push their branch up to the remote repo and do a pull request to master, which another team member would review and then merge, after resolving any conflicts.
 
+A good way to avoid merge conflicts is to do a pull from your remote master, and merge any changes into your branch before pushing up the branch to the remote repo.
```

Comment:
:+1: 
This last step is important, but it won't necessarily help you avoid merge conflicts.  It usually makes them less painful.